simphony_ui/couple_CFDDEM.py

```diff
@@ -7,22 +7,18 @@
 import time
 import math
 import numpy as np
-import subprocess
 import OpenFoam_input
 import tempfile
-import simliggghts
```

Comment:
Sounds like it isn't. Is it weird? We only import the CUBAExtension from simliggghts.